modules/nw-modifying-operator-install-config.adoc

```diff
@@ -55,22 +55,9 @@ metadata:
 spec:
 ----
 
-. Specify the advanced network configuration for your cluster in the `cluster-network-03-config.yml` file, such as in the following examples:
+. Specify the advanced network configuration for your cluster in the `cluster-network-03-config.yml` file, such as in the following example:
 +
 --
-.Specify a different VXLAN port for the OpenShift SDN network provider
-[source,yaml]
-----
-apiVersion: operator.openshift.io/v1
-kind: Network
-metadata:
-  name: cluster
-spec:
-  defaultNetwork:
-    openshiftSDNConfig:
-      vxlanPort: 4800
-----
-
 .Enable IPsec for the OVN-Kubernetes network provider
```

Comment:
@anuragthehatter Enabling IPsec has different modes now. Do we need more details here?

Reply:
I believe there will revisit of this field, can ignore for now.